src/pages/Home/Home.page.tsx

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 
 import Slider from '../../components/Slider';
 import Categories from '../../components/Categories';
 import Products from '../../components/Products';
+import featuredProducts from '../../mocks/en-us/featured-products.json';
+import { Product } from '../../utils/types';
+import { HomeWrapper, Button } from './Home.styled';
 
 function HomePage() {
+  const { push } = useHistory();
+  const [featProducts, setFeatProducts] = useState<Product[]>([]);
+
+  useEffect(() => {
```

Comment:
Is this useEffect needed? Or we can initialize the useState variable by default?

Reply:
Done... thank you for the recommendation!